### DISCUSSION TITLE
domcloud in youtube and blogs

### DISCUSSION THREAD

**Discussion by elycheikhsmail** [domcloud/domcloud#40] [Ideas]
even domcloud is good solutions for small and midium web sites, I don't any video on youtube explaining about domcloud, also I don't see articles about it.

**Comment by willnode** [comment]
Good question.

I don't spend money on marketing. We're growing organically. That's also the reason why it can be generous. The only magnet that spikes the traffic is I listed this platform on [free-for.dev](https://free-for.dev/#/) following Heroku's decision to sunset their free plan.

I don't create tutorials yet because I must first stabilize the platform. You can help by creating tutorials if you like. That really means something to us.

**Reply by elycheikhsmail** [reply]
I will be happy to create content about domcloud but I must take some time to test it by using it in different contexts.
what you think is missing in domcloud to be considere as stable ?

**Reply by willnode** [reply]
Documentation and GitHub integration need major improvements
